### PR TITLE
OCRVS-83: replaced CBS references

### PR DESCRIPTION
**Pull request by kaweesi**

### CODE REVIEW

File: app/config/default.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "protocol": "http",
   "host": "localhost",
-  "port": 5001
-}
\ No newline at end of file
+  "port": 3447
```
Comment (kaweesi) [inline app/config/default.json]: i should have ignored this change

**Comment by napiergit** [comment]
@kaweesi there are still a couple of rcbs references

image

**Comment by napiergit** [comment]
@kaweesi looking better, unfortunately I still see a few references to cbs.  Also, can you please rename opencrvs to ocrvs as discussed on Carbon chat earlier.

**Comment by kaweesi** [comment]
there are things that are CBS related that i am leaving since i am thinking we are soon separating these from the repo like; http://hearth.org/cbs/event-types etc

**Comment by napiergit** [comment]
Cool @kaweesi looks good to me.  Just a merge conflict

File: app/config/default.json (hunk 1 above)
Comment (BMartinos) [inline app/config/default.json]: I think this needs to be reverted back to 5001 for routing the transactions through the OpenHIM

**Pull request merged**